logger/logger.go

```diff
@@ -6,90 +6,37 @@ import (
 	"strings"
 	"time"
 
-	std_log "log"
-
-	go_log "git.sr.ht/~spc/go-log"
+	logrus "github.com/sirupsen/logrus"
 )
 
-const defaultLogFormat = 0
-const defaultLogPrefix = ""
-
 const (
-	DebugLevel = "DEBUG"
-	InfoLevel  = "INFO"
-	WarnLevel  = "WARN"
-	ErrorLevel = "ERROR"
-	TraceLevel = "TRACE"
```

Comment:
The FieldLogger interface for Logrus doesn't implement the Trace level, but this level wasn't used anywhere in the host-metering code so it can be removed without any breaking changes